src/test/java/tests/JsonHelperTest.java

```diff
@@ -0,0 +1,191 @@
+package tests;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import util.JsonHelper;
+
+import java.util.Arrays;
+
+public class JsonHelperTest {
+
+    /**
+     * Dummy class to test json conversion
+     */
+    private static class Dummy {
+        // default values
+        private int testInt = 3;
+        private boolean testBoolean = false;
```

Comment:
boolean naming convention. I shouldn't have to remind of this more than once. :-p